### DISCUSSION TITLE
Support for Lerna default symlinks in pages/

### DISCUSSION THREAD

**Discussion by nicholaschiang** [vercel/next.js#11641] [Help]
# Bug report

## Describe the bug

[Similar issues](https://github.com/zeit/next.js/issues/3728) have been extensively discussed, but I haven't found a concise answer for my particular quandary yet.

I am managing [this Next.js project](https://github.com/tutorbookapp/covid-tutoring) and want to use [Lerna](https://lerna.js.org) to keep each component as a separate `npm` package within the `src/` directory:
```
.                      
├── form
│   ├── index.module.scss
│   ├── index.tsx
│   ├── node_modules
│   │   └── @tutorbook
│   │       ├── animated-checkmark-overlay -> ../../../animated-checkmark-overlay
│   │       ├── arrow-btn -> ../../../arrow-btn
│   │       ├── covid-fonts -> ../../../fonts
│   │       └── spinner -> ../../../spinner
│   ├── package.json
│   └── package-lock.json
└── pages       
     ├── about.tsx   
     ├── _app.tsx    
     ├── index.tsx        
     ├── node_modules
     │   └── @tutorbook
     │       ├── covid-about -> ../../../about
     │       ├── covid-footer -> ../../../footer
     │       ├── covid-header -> ../../../header
     │       ├── covid-pupil-form -> ../../../pupil-form
     │       ├── covid-styles -> ../../../styles
     │       ├── covid-tutor-form -> ../../../tutor-form
     │       └── next-firebase -> ../../../firebase
     ├── package.json                         
     ├── package-lock.json
     ├── pupils.tsx       
     └── tutors.tsx
```

## Expected behavior

We should be able to ignore

## Screenshots

Those Lerna-generated symlinks have some issues though as users can access the individual components:

![image](https://user-images.githubusercontent.com/20798889/78394579-bd567b00-75a0-11ea-85ce-e86dfe324e3d.png)

## System information

- OS: Ubuntu 18.04.2
- Version of Next.js: `^9.3.4`

**Comment by nicholaschiang** [comment]
Current (very bad) workaround is to include them in the Zeit build command:
```
$ lerna bootstrap --hoist && rm -r src/pages/node_modules/ && npm run build
```

**Comment by timneutkens** [comment]
I don't understand why you have a `package.json` and `node_modules` in the pages directory 🤔 Looking at the description I don't see anything that seems like a bug in Next.js so I've converted this to a discussion.
